tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -431,64 +460,48 @@ impl<'a> field::Visit for PrettyVisitor<'a> {
             return;
         }
 
-        if field.name() == "message" {
-            self.record_debug(field, &format_args!("{}", value))
+        self.result = if field.name() == "message" {
+            self.record_debug_impl(field, &format_args!("{}", self.style.paint(value)))
         } else {
-            self.record_debug(field, &value)
+            self.record_debug_impl(field, &self.style.paint(value))
```

Comment:
Is there a difference between the two calls? I wasn't sure, so I was afraid of touching this code.